docs/pages/reference/mesheryctl/commands/mesheryctl_command_template.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: Command Doc Title

```

Comment:
```suggestion
title: mesheryctl command name
```

Reply:
or `mesheryctl <command name>` if that can be added. It would be better so the contributors will know that this should be changed.